plugins/pastefromwordimage/plugin.js

```diff
@@ -0,0 +1,99 @@
+/**
+ * @license Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md or http://ckeditor.com/license
+ */
+
+( function() {
+	'use strict';
+
+	CKEDITOR.plugins.add( 'pastefromwordimage', {
+		requires: 'pastefromword',
+		init: function() {}
+	} );
+
+	/**
+	 * Help methods used by paste from word image plugin.
+	 *
+	 * @since 4.8.0
+	 * @class CKEDITOR.plugins.pastefromwordimage
+	 */
+	CKEDITOR.plugins.pastefromwordimage = {
+		/**
+		 * Methods parse RTF clipboard to find embedded images.
+		 *
+		 * @private
+		 * @since 4.8.0
+		 * @param {String} rtfClipboard Data obtained from RTF clipboard.
+		 * @returns {Array} Contains array of objects with images or empty array if there weren't a match.
+		 * @returns {Object} return.Object Single image found in `rtfClipboard`.
```

Comment:
Hmmm it seems to work quite well and docs are build without any error. I can't find information how to represent return value in such case (array of objects with properties description). Do you have any suggestions how it might be described? Or should I just remove all information what is returned by this method?
![screenshot-localhost-2017-10-02-13-51-17-800](https://user-images.githubusercontent.com/9017322/31076131-2214390e-a779-11e7-8c0e-4a4ac862e5b7.png)